packages/x-charts/src/ChartsTooltip/DefaultChartsAxisTooltipContent.tsx

```diff
@@ -18,7 +18,18 @@ function DefaultChartsAxisTooltipContent(props: ChartsAxisContentProps) {
   if (dataIndex == null) {
     return null;
   }
-  const axisFormatter = axis.valueFormatter ?? ((v) => v.toLocaleString());
+
+  const utcFormatter = (v: string | number | Date): string => {
+    if (v instanceof Date) {
+      return v.toUTCString();
+    }
+    return v.toLocaleString();
+  };
+
+  const axisFormatter =
+    axis.valueFormatter ??
+    ((v: any) => (axis?.scaleType === 'utc' ? utcFormatter(v) : v.toLocaleString()));
```

Comment:
Why the `?` in `axis?.scaleType`?

Can we avoid the `any`? Maybe by inverting the order of call: 
```
axis?.scaleType === 'utc' ? utcFormatter : (v) => v.toLocaleString();
```